Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -30,6 +30,8 @@ struct FPTIBatchData: Codable {
         let correlationID: String?
         let errorDescription: String?
         let eventName: String
+        /// Checks whether Merchant launched tokenzie request with Vault
```

Comment:
Nitpick, take or leave
```suggestion
        /// True if `tokenize()` was called with a VaultRequest object type
```